apps/public-docsite-v9/src/Concepts/Migration/KeepingDesignConsistent.stories.mdx

```diff
@@ -0,0 +1,147 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Concepts/Migration/Keeping Design Consistent" />
+
+# Keeping Design Consistent
+
+Fluent 2 is the next version of the Fluent design language.
+Fluent UI React v9 uses the Fluent 2 design language for layout and style of components.
+
+The design language defines a set of design tokens.
+Themes consist of a property/value pair for each token.
+Designers reference these tokens in the design specs detailing the layout and style of components.
+
+The themes in v0 and v8 were specific to component, component part, and state.
+While this allowed for fine-grained control over each component's style, it led to an explosion of theme properties.
+Too many theme properties can make defining new themes an arduous chore, can make themes fragile when properties are added or removed, and can leave dead theme properties behind when components stop using them.
+
+In v9, the theme properties are called design tokens.
+They are much more general purpose.
+They are partitioned by neutral vs. brand, usage (e.g. foreground, background, stroke), and state.
+
+This topic covers ways you can maintain a consistent theme and style during migration.
+
+## You can choose to live with and limit style differences during migration
+
+The visual style differences between v9 and previous versions are slight.
+If your migration effort is small, you can migrate all at once, or your users are OK with some inconsistency, you might decide to avoid extra effort and live with it.
```

Comment:
```suggestion
If your migration effort is small, you can migrate all at once, or if your users are OK with some inconsistency, you might decide to avoid extra effort and live with it.
```